src/wix/WixToolset.Core.Burn/Bundles/BurnCommon.cs

```diff
@@ -19,7 +20,6 @@ internal abstract class BurnCommon : IDisposable
     {
         public const string BurnNamespace = "http://wixtoolset.org/schemas/v4/2008/Burn";
         public const string BurnUXContainerEmbeddedIdFormat = "u{0}";
-        public const string BurnAuthoredContainerEmbeddedIdFormat = "a{0}";
```

Comment:
Need to add a test to verify that either:
* This is change is required to prevent collisions or
* This change is not required, so it can be reverted along with the test changes